### PR TITLE
💄UI-UX Design - Adding functionality to add links to Case Studies and Videos 

### LINKED ISSUE #1013: ✨ Consulting - Add Case Study + Video to UI-UX-Design page
URL: https://www.ssw.com.au/consulting/ui-ux-design

**Description:**

See https://ssw.com.au/ssw/Company/Clients/
Add all Case Study links + Videos to the respective client to UI/UX design page

### PR DESCRIPTION
**Pull request by amankumarrr**
Affected routes: 
- `/consulting/ui-ux-design`

- Fixed #1013 

Screenshot:

image

**Figure: Case Study and Video button with the company's carousel**

### CODE REVIEW

**Review by tiagov8 — APPROVED** [review]

File: tailwind.config.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ export const platform = [
 ];
 
 /** @type {import("tailwindcss").Config} */
+// eslint-disable-next-line import/no-anonymous-default-export
 export default {
   // mode: "jit",
```
Comment (Calinator444) [inline tailwind.config.js:53]: I think this is commented code. I'd probably remove it.

Comment (Calinator444) [inline tailwind.config.js:52]: This looks like commented code, I'd probably remove it. Other than that LGTM.

Comment (amankumarrr) [reply tailwind.config.js:52]: Without it, the code editor displays a multitude of yellow squiggles, making it difficult to identify and update specific areas. Let me know what do you think?

image

Comment (Calinator444) [reply tailwind.config.js:52]: I was referring to the `// mode: jit,` line. The linting configuration is fine :)

Comment (amankumarrr) [reply tailwind.config.js:52]: oh right! Actually with comment this line is highlighted
image

Comment (Calinator444) [reply tailwind.config.js:52]: the commented code might not have been a file change so when I clicked it added the comment to the highlighted change instead

Comment (amankumarrr) [reply tailwind.config.js:52]: done

**Review by Calinator444 — APPROVED** [review]

**Pull request merged**